build/commands/lib/config.js

```diff
@@ -737,6 +739,8 @@ Config.prototype.buildArgs = function () {
     delete args.enable_glic
     delete args.enable_widevine
     delete args.enable_hangout_services_extension
+    delete args.brave_account_api_endpoint
```

Comment:
I don't think we need these per above, but just curious why you removed them for ios. This will work on ios as well, right?